features/admin.roles.v2/components/role-utils.ts

```diff
@@ -51,3 +51,21 @@ export const generatePermissionTree = (permissioObject: PermissionObject, pathCo
 
     return permissionTreeArray;
 };
+
+/**
+ * Checks if the given role is an impersonation role for the My Account application.
+ *
+ * @param roleName - The name of the role.
+ * @param applicationName - The name of the application.
+ *
+ * @returns True if the role is an impersonation role for My Account application, false otherwise.
+ */
+export const isMyAccountImpersonationRole = (roleName:string, applicationName:string): boolean => {
+
+    if (!roleName || !applicationName) {
+        return false;
+    }
+
+    return roleName === RoleConstants.IMPERSONATOR_ROLE_NAME
+        && applicationName === RoleConstants.MY_ACCOUNT_APP_NAME;
```

Comment:
It doesn't make sense to take my account app name from role constants IMO

Reply:
Ideally we should take myaccount app name from deployment config so that end users will be able to customize myaccount app name through deployment.toml